data/json/items/armor/jewelry.json

```
@@ -92,7 +88,7 @@
     "color": "yellow",
     "covers": [ "mouth" ],
     "encumbrance": 10,
-    "flags": [ "SUPER_FANCY", "SKINTIGHT" ]
+    "flags": [ "SKINTIGHT" ]
```

```suggestion
    "flags": [ "FANCY", "SKINTIGHT" ]
```
Is it black tie? No. Is it something that someone wears to feel like they're special and impress people? Yes.

---

The goal *is* "black tie", it needs to be relatively exclusive, otherwise it's pointless.